pkg/cli/tsdump.go

```diff
@@ -60,6 +61,21 @@ var debugTimeSeriesDumpOpts = struct {
 	yaml:         "/tmp/tsdump.yaml",
 }
 
+var (
+	// each site in datadog has a different host name. ddSiteToHostMap
+	// holds the mapping of site name to the host name.
+	ddSiteToHostMap = map[string]string{
```

Comment:
I think I also added the same map. Since this will get merged first, I will remove the map from zip_upload.go and reuse this.